packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -528,6 +536,56 @@ export class DefaultCrudRepository<
     return includeRelatedModels<T, Relations>(this, entities, include, options);
   }
 
+  protected async entityToData<R extends T>(
+    entity: R | DataObject<R>,
+    options = {},
+  ): Promise<legacy.ModelData<legacy.PersistedModel>> {
+    this.ensurePersistable(entity, options);
+    return entity;
```

Comment:
`this.ensurePersistable` is returning a new data object. Should we use it instead of returning `entity`?

Reply:
👍 fixed